fiber-opentelemetry/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/lightstep/otel-launcher-go/launcher"
+	fiberOtel "github.com/psmarcin/fiber-opentelemetry/pkg/fiber-otel"
+)
+
+func main() {
+	// configuration
+	// collector setup is out of the scope of this example so we don't focus on it, just go to lightstep.com
```

Comment:
Please give more instructions for each config points for users.

Reply:
I think setting up open-telemetry connections and configuration is not a scope of `fiber-opentelemetry`, so I don't think it needs to be documented at all.